src/bournemouth/resources.py

```
@@ -24,10 +25,15 @@
     get_or_create_conversation,
     list_conversation_messages,
     load_user_and_api_key,
-    stream_answer,
+)
+from .chat_utils import (
+    ChatWsRequest,
+    ChatWsResponse,
+    build_chat_history,
+    stream_chat_response,
 )
```

Pylint (3.3.7)

[convention] Import "from .chat_utils import ChatWsRequest, ChatWsResponse, build_chat_history, stream_chat_response" should be placed at the top of the module

(C0413)